config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
/boot/<bootloader> is a mountpoint, like for uboot, efi or grub we need to create that dir during core snap build time. the initramfs needs to care for the mounting like it does for all the other bootloaders (see: https://github.com/snapcore/core-build/blob/master/initramfs/scripts/ubuntu-core-rootfs#L315 )